src/App.js

```diff
@@ -21,24 +21,38 @@ function App() {
   return (
     <>
       <ThemeProvider theme={theme}>
-        <Route exact path="/" component={Dashboard} />
-        <Route exact path="/success" component={Success} />
-        <Route exact path="/success-screen-two" component={SuccessScreenTwo} />
-        <Route exact path="/scheduledays" component={ScheduleAvailableDays} />
-        <Route exact path="/scheduletimes" component={ScheduleAvailableTimes} />
-        <Route exact path="/meetingtypes" component={MeetingTypes} />
-        <Route exact path="/preferences" component={Preferences} />
-        <Route exact path="/prompt" component={Prompt} />
-        <Route exact path="/review-preferences" component={ReviewPreferences} />
-        <Route exact path="/calendarview" component={CalendarView} />
-        <Route exact path="/contactreview" component={ContactReview} />
-        <Route exact path="/hostsuccess" component={HostSuccess} />
-        <Route exact path="/guestsuccess" component={GuestSuccess} />
-        <Route
-          exact
-          path="/calendarselectedtimes"
-          component={CalendarSelectedTimes}
-        />
+        <main style={{ maxWidth: 812, margin: '0 auto' }}>
```

Comment:
We should have a `main` tag somewhere in our app. I'm not sure if this is the best place to put it, but I threw it in here for now. 

Also, it's a little hacky, but it was bothering me how the app looks so bad when not on mobile, so I just kind of hard coded a `maxWidth` for the entire app.  Not a great solution.

Reply:
Yeah, considering how the app looks when not on mobile, are we going to have a design for how the app's web version is going to look like?

Reply:
I heard a long time ago that a desktop version is in the works, but I haven't heard anything about it in a while.

Reply:
Oh okay sounds good! I'll be waiting until then.